src/utils/validateUtil.ts

```diff
@@ -181,7 +181,7 @@ export function validateRules(
               callback();
             })
             .catch(err => {
-              callback(err);
+              callback(err || rule.message || options.validateMessages?.default || 'error');
```

Comment:
Should check if validateMessages with variable work like: `Field '${name}' is wrong`.

Reply:
I haven't understood how to use `validateMessages` at the moment.

Reply:
Maybe should extends `validateRules` function to provides additional `validateMessages` for templating.